cfpb/cfgov-refresh
# Update optional form input style

Required/optional fields were inconsistently styled. See https://[GHE]/CFGOV/platform/issues/465. This makes them consistent.

## Changes

- Remove asterisk for required fields on conference registration form.
- Makes optional fields have `(optional)` text next to them.

## Testing

1. Create a new browse page in Wagtail.
2. Add the conference registration form.
3. Preview the page and view the optional/required fields.
4. Change https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_includes/conference-registration-form.html#L99 to false to see checkbox optional style.

## Screenshots

Before:

<img width="915" alt="screen shot 2017-08-04 at 5 54 00 pm" src="https://user-images.githubusercontent.com/704760/28988576-6ab6b70e-793e-11e7-96cc-817b9c3aff46.png">

After:
<img width="942" alt="screen shot 2017-08-04 at 5 53 17 pm" src="https://user-images.githubusercontent.com/704760/28988577-6abe3e34-793e-11e7-981f-4fcfce9b2be1.png">

## Notes

- Checkbox background is messed up, but that was that way outside of this PR.
